x-pack/plugins/enterprise_search/server/routes/workplace_search/sources.ts

```diff
@@ -743,6 +753,30 @@ export function registerOrgSourceOauthConfigurationsRoute({
       path: '/ws/org/settings/connectors',
     })
   );
+
+  router.post(
+    {
+      path: '/api/workplace_search/org/settings/connectors',
+      validate: {
+        body: oAuthConfigSchema,
+      },
+    },
+    enterpriseSearchRequestHandler.createRequest({
+      path: '/ws/org/settings/connectors',
+    })
+  );
+
+  router.put(
+    {
+      path: '/api/workplace_search/org/settings/connectors',
+      validate: {
+        body: oAuthConfigSchema,
```

Comment:
In other places we call capitalize it differently: `Oauth` instead of `oAuth`. I think it can be considered a single word, so let's rename this variable to `Oauth...`?